### PR TITLE
Align MCP gateway image and hub with ASM unmanaged

### PR DESCRIPTION
**Pull request by ruigulala**

### CODE REVIEW

**Review by zerobfd — APPROVED** [review]

**Review by stewartbutler — APPROVED** [review]

**Review by costinm — APPROVED** [review]

File: asm/istio/options/managed-control-plane.yaml [hunk 1]
```diff
@@ -18,8 +18,8 @@ kind: IstioOperator
 spec:
   revision: "asm-managed"
   profile: asm-gcp
-  hub: gcr.io/wlhe-cr/asm
-  tag: 1.9-alpha.1612961842
+  hub: gcr.io/gke-release/asm # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.hub"}
+  tag: 1.8.2-asm.2 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.tag"}
```
Comment (costinm) [inline asm/istio/options/managed-control-plane.yaml:22]: BTW - all defaults should be set to 1.9.0-asm.1 ( AFAIK ) before Tue

Comment (stewartbutler) [reply asm/istio/options/managed-control-plane.yaml:22]: I imagine that'll be handled in one batch commit across the entire repo.

Comment (ruigulala) [reply asm/istio/options/managed-control-plane.yaml:22]: Yeah, it's currently aligned with the rest of the repo.
https://github.com/GoogleCloudPlatform/anthos-service-mesh-packages/blob/master/asm/istio/istio-operator.yaml#L20-L21

**Pull request merged**